Hearthstone Deck Tracker/FlyoutControls/Options/Tracker/TrackerPlugins.xaml.cs

```diff
@@ -60,5 +62,40 @@ private void ButtonOpenPluginsFolder_OnClick(object sender, RoutedEventArgs e)
 			}
 			Helper.TryOpenUrl(dir.FullName);
 		}
+
+		private async void GroupBox_Drop(object sender, DragEventArgs e)
+		{
+			var dir = PluginManager.PluginDirectory.FullName;
+			int plugins = 0;
+			try
+			{
+				if(e.Data.GetDataPresent(DataFormats.FileDrop))
+				{
+					string[] droppedFiles = (string[])e.Data.GetData(DataFormats.FileDrop);
+					if (droppedFiles == null) return;
```

Comment:
`return` and `continue` (below) should go on new lines,
No space after `if` please (same below).